data_import/signal_helpers.py

```diff
@@ -5,6 +5,55 @@
 from .models import DataRetrievalTask
 
 
+def task_signal_pre_save(task_params, datafile_model, **kwargs):
+    """
+    Trigger data retrieval a study adds new data via UserData's data field.
+
+    Data retrieval task creation is triggered on a pre_save signal to a
+    UserData object (derived from studies.models.BaseStudyUserData) and only
+    occurs if the new instance's data field is non-empty and changed.
+
+    If the user's email address is verified, the task is started. Otherwise it
+    is postponed.
+    """
+    instance = kwargs['instance']
+
+    # Skip is this is fixture data.
+    if kwargs['raw']:
```

Comment:
any kwarg that we always use could be added to the method parameters directly (so this one and `sender`)